src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -39,6 +40,23 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             info.AddValue("Exceptions", LoaderExceptions, typeof(Exception[]));
         }
 
+        /// <summary>
+        /// Creates and returns a string representation of the current <see cref="AggregateException"/>.
+        /// </summary>
+        /// <returns>A string representation of the current exception.</returns>
+        public override string ToString()
```

Comment:
Is it sufficient to override ToString to get what you need in all places? AggregateException overrides Message property as well.

Reply:
> Is it sufficient to override ToString to get what you need in all places? AggregateException overrides Message property as well.

I didn't notice that, or I would have suggested it. Yes please let's make Message do the same thing as ToString, but without the callstack/s.